### PR TITLE
21204 - Create migration that add EFT and PAD into suspension_reason_codes

### PR DESCRIPTION
**Pull request by sameer0422**
*Issue #:*
https://github.com/bcgov/entity/issues/21204

*Description of changes:* Create migration that updates suspension_reason_code

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2844-em9wmvys.web.app

File: auth-api/migrations/versions/2024_05_29_41fa6588c76e_add_eft_pad_to_suspension_reason_code.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+down_revision = '1e4b6359f470'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute("INSERT INTO suspension_reason_codes "
+            "(code, description,\"default\") "
+            "VALUES "
+            "('EFT', 'Electronic Funds Transfer', false)")
```
Comment (Jxio) [inline auth-api/migrations/versions/2024_05_29_41fa6588c76e_add_eft_pad_to_suspension_reason_code.py:23]: EFT is a type of payment method, not suspension_reason_code.
Instead I think we can use 'EFT_NSF', 'EFT NSF'

File: auth-api/migrations/versions/2024_05_29_41fa6588c76e_add_eft_pad_to_suspension_reason_code.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    op.execute("INSERT INTO suspension_reason_codes "
+            "(code, description,\"default\") "
+            "VALUES "
+            "('PAD', 'Pre-Authorized Debit', false)")
```
Comment (Jxio) [inline auth-api/migrations/versions/2024_05_29_41fa6588c76e_add_eft_pad_to_suspension_reason_code.py:27]: PAD is a type of payment method, not suspension_reason_code.

Comment (sameer0422) [reply auth-api/migrations/versions/2024_05_29_41fa6588c76e_add_eft_pad_to_suspension_reason_code.py]: Fixed

File: auth-api/migrations/versions/2024_05_29_41fa6588c76e_add_eft_pad_to_suspension_reason_code.py (hunk 1 above)
Comment (sameer0422) [reply auth-api/migrations/versions/2024_05_29_41fa6588c76e_add_eft_pad_to_suspension_reason_code.py]: Fixed

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2844-em9wmvys.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-account-dev--pr-2844-em9wmvys.web.app

**Review by Jxio — APPROVED** [review]
LGTM!

**Pull request merged**